en_US/durability/managing-replication.md

```diff
@@ -1,44 +1,44 @@
-# Replication Management
+# Manage Replication
 
-This document describes how to setup data replication and high availability for the EMQX durable storage.
-It is relevant when setting up a new EMQX cluster with session persistence enabled, or upgrading an existing cluster to enable session persistence.
+This guide provides instructions on configuring data replication and ensuring high availability for EMQX durable storage. This guide consists of instructions for two scenarios: setting up a new EMQX cluster with durable storage and upgrading an existing cluster to enable durable storage.
```

Comment:
> This guide provides

This document is not meant as a guide. It's meant to be a rather in-depth technical overview of the replication process, that answers most of the questions for the technically inclined users.